admin/banner_manager.php

```diff
@@ -351,127 +351,127 @@ function popupImageWindow(url) {
-              <span class="help-block"><?php echo TEXT_INFO_BANNER_ON_SSL; ?></span>
+              <label class="radio-inline"><?= zen_draw_radio_field('banners_on_ssl', '1', $bInfo->banners_on_ssl == 1) . TEXT_YES ?></label>
+              <label class="radio-inline"><?= zen_draw_radio_field('banners_on_ssl', '0', $bInfo->banners_on_ssl == 0) . TEXT_NO ?></label><br>
+              <span class="help-block"><?= TEXT_INFO_BANNER_ON_SSL ?></span>
             </div>
           </div>
           <div class="form-group">
-            <?php echo zen_draw_label(TEXT_BANNERS_TITLE, 'banners_title', 'class="col-sm-3 control-label"'); ?>
+            <?= zen_draw_label(TEXT_BANNERS_TITLE, 'banners_title', 'class="col-sm-3 control-label"') ?>
             <div class="col-sm-9 col-md-6">
-              <?php echo zen_draw_input_field('banners_title', htmlspecialchars($bInfo->banners_title, ENT_COMPAT, CHARSET), zen_set_field_length(TABLE_BANNERS, 'banners_title') . ' class="form-control" id="banners_title"', true); ?>
+              <?= zen_draw_input_field('banners_title', htmlspecialchars($bInfo->banners_title, ENT_COMPAT, CHARSET), zen_set_field_length(TABLE_BANNERS, 'banners_title') . ' class="form-control" id="banners_title"', true) ?>
             </div>
           </div>
           <div class="form-group">
-            <?php echo zen_draw_label(TEXT_BANNERS_URL, 'banners_url', 'class="col-sm-3 control-label"'); ?>
+            <?= zen_draw_label(TEXT_BANNERS_URL, 'banners_url', 'class="col-sm-3 control-label"') ?>
             <div class="col-sm-9 col-md-6">
-              <?php echo zen_draw_input_field('banners_url', $bInfo->banners_url, zen_set_field_length(TABLE_BANNERS, 'banners_url') . ' class="form-control" id="banners_url"'); ?>
+              <?= zen_draw_input_field('banners_url', $bInfo->banners_url, zen_set_field_length(TABLE_BANNERS, 'banners_url') . ' class="form-control" id="banners_url"') ?>
             </div>
           </div>
           <div class="form-group">
-            <?php echo zen_draw_label(TEXT_BANNERS_GROUP, 'banners_group', 'class="col-sm-3 control-label"'); ?>
+            <?= zen_draw_label(TEXT_BANNERS_GROUP, 'banners_group', 'class="col-sm-3 control-label"') ?>
             <div class="col-sm-9 col-md-6">
-              <?php echo zen_draw_pull_down_menu('banners_group', $groups_array, $bInfo->banners_group, 'class="form-control" id="banners_group"'); ?><br>
-              <p><?php echo TEXT_BANNERS_NEW_GROUP; ?></p><?php echo zen_draw_input_field('new_banners_group', '', 'class="form-control" id="new_banners_group"', count($groups_array) === 0); ?>
+              <?= zen_draw_pull_down_menu('banners_group', $groups_array, $bInfo->banners_group, 'class="form-control" id="banners_group"') ?><br>
+              <p><?= TEXT_BANNERS_NEW_GROUP; ?></p><?php echo zen_draw_input_field('new_banners_group', '', 'class="form-control" id="new_banners_group"', count($groups_array) === 0) ?>
```

Comment:
Looks like the regex pattern missed cases where multiple exist on the same line.